### PR TITLE
Release 2.12.0

### PR DESCRIPTION
**Pull request by sven1103**
**Added**

* Add ``life.qbic.datamodel.validation.*`` for project code validation. #254
* ``life.qbic.datamodel.dtos.business.facilities.Facility`` can provide a shorthand label now #255 

**Fixed**

**Dependencies**

**Deprecated**

### CODE REVIEW

File: .qube.yml [hunk 1]
```diff
@@ -4,7 +4,7 @@ email: sven.fillinger@qbic.uni-tuebingen.de
 project_name: data-model-lib
 project_short_description: "Data models. A collection of QBiC's central data models\
   \ and DTOs. "
-version: 2.11.0
+version: 2.12.0-SNAPSHOT
```
Comment (jenniferboedker) [inline .qube.yml:7]:
```suggestion
version: 2.12.0
```

File: docs/conf.py [hunk 2]
```diff
@@ -55,9 +55,9 @@
 # the built documents.
 #
 # The short X.Y version.
-version = '2.11.0'
+version = '2.12.0-SNAPSHOT'
```
Comment (jenniferboedker) [inline docs/conf.py:58]:
```suggestion
version = '2.12.0'
```

File: docs/conf.py [hunk 3]
```diff
@@ -55,9 +55,9 @@
...
 # The full version, including alpha/beta/rc tags.
-release = '2.11.0'
+release = '2.12.0-SNAPSHOT'
```
Comment (jenniferboedker) [inline docs/conf.py:60]:
```suggestion
release = '2.12.0'
```

File: pom.xml [hunk 4]
```diff
@@ -7,7 +7,7 @@
 
     <groupId>life.qbic</groupId>
 	<artifactId>data-model-lib</artifactId>
-	<version>2.11.0</version> <!-- <<QUBE_FORCE_BUMP>> -->
+	<version>2.12.0-SNAPSHOT</version> <!-- <<QUBE_FORCE_BUMP>> -->
```
Comment (jenniferboedker) [inline pom.xml:10]:
```suggestion
	<version>2.12.0</version> <!-- <<QUBE_FORCE_BUMP>> -->
```

File: qube.cfg [hunk 5]
```diff
@@ -1,5 +1,5 @@
 [bumpversion]
-current_version = 2.11.0
+current_version = 2.12.0-SNAPSHOT
```
Comment (jenniferboedker) [inline qube.cfg:2]:
```suggestion
current_version = 2.12.0
```

**Review by jenniferboedker — CHANGES_REQUESTED** [review]
Looks good, though you need to bump the version to 2.12.0

File: pom.xml (hunk 4 above)
Comment (sven1103) [reply pom.xml:10]: Maven does this for us automatically :)

**Review by KochTobi — APPROVED** [review]

File: pom.xml (hunk 4 above)
Comment (jenniferboedker) [reply pom.xml:10]: ahh damn, I forgot that this is done for the OM ..

**Review by jenniferboedker — APPROVED** [review]
🚀

**Pull request merged**